jdbc-adapter/doc/db2-dialect.md

```diff
@@ -0,0 +1,18 @@
+# DB2 Dialect
+Short description of DB2 Dialect functionality and features
+
+## Casting of Data Types
+* TIMESTAMP and TIMESTAMP(x) will be cast to VARCHAR to not lose precision.
+* VARCHAR and CHAR for bit data will be cast to a hex string with double the original size
+* TIME will be cast to VARCHAR(8)
+* XML will be cast to VARCHAR(DB2_MAX_LENGTH)
+* BLOB is not supported
+
+## Casting of Functions
+* LIMIT will replaced by FETCH FIRST x ROWS ONLY
+* OFFESET currently not supported as only DB2 V11 support this
+* ADD_DAYS, ADD_WEEKS ... will be replaced by COLUMN + DAYS, COLUMN + ....
+
+## Unit Tests 
```

Comment:
Although the tests are written using junit we refer to them as integration test in the documentation, because they test the whole system and run in the integration-test maven phase. There are only a few real unit tests ("Test" suffix in filename inestead of "IT" suffix), but most of the dialects are tested with integration tests.